include/deal.II/matrix_free/portable_fe_evaluation.h

```diff
@@ -270,6 +286,38 @@ namespace Portable
 
 
 
+  template <int dim,
+            int fe_degree,
+            int n_q_points_1d,
+            int n_components_,
+            typename Number>
+  DEAL_II_HOST_DEVICE int
+  FEEvaluation<dim, fe_degree, n_q_points_1d, n_components_, Number>::
+    get_current_cell_index()
```

Comment:
We probably want to inline these functions.

Reply:
Can I do that in a follow-up? After the planned restructuring, they have to change anyways.

Reply:
Sure